Replacing bash brace expansion with a Posix compliant syntax to make
do_install succeed also when using a dash shell.

Upstream-Status: Pending

Signed-off-by: Max Krummenacher <max.oss.09@gmail.com>
--- libmcc-1.04/Makefile.orig	2014-08-09 11:17:24.006935887 +0200
+++ libmcc-1.04/Makefile	2014-08-09 11:19:36.867546971 +0200
@@ -11,8 +11,8 @@
 	$(CC) -shared -o $@  $<
 
 install:
-	mkdir -p $(DESTDIR)/usr/{lib,include}
-	cp -f build/libmcc.{so.*,a} $(DESTDIR)/usr/lib
+	mkdir -p $(DESTDIR)/usr/include $(DESTDIR)/usr/lib
+	cp -f build/libmcc.so.* build/libmcc.a $(DESTDIR)/usr/lib
 	cp -f include/*.h $(DESTDIR)/usr/include
 	cd $(DESTDIR)/usr/lib/
 	ln -sf libmcc.so.1.0 libmcc.so.1
